### DISCUSSION TITLE
Sort order with multiple printers

### DISCUSSION THREAD

**Discussion by Chron-O-John** [KlipperScreen/KlipperScreen#1194] [Q&A]
Hi,
Did I miss it, or is there currently no way to set a (sort-)order for the selection when multiple printers are configured?

**Comment by alfrix** [comment]
Currently there is no way to change sorting.

a quick change would be to just remove the default sort so the printers will appear in the order defined in the config, if that is the case please make a feature request

**Comment by alfrix** [comment]
you should be able to sort them by order of appearance in the config [1f43391](https://github.com/KlipperScreen/KlipperScreen/commit/1f43391b2cbd55cc5551544c74b1bebd968ee53f)
